extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -11,7 +11,10 @@ class WikiaMapsMapController extends WikiaMapsBaseController {
 	 * @return Array
 	 */
 	public function getTileSets() {
-		$params = [];
+		$params = [
```

Comment:
IMO the reason to write unit test here would be to 'freeze' values of parameters 'sort' and 'limit'. But after talking to Rafał I believe there is no reason to do that - as the service back-end will handle it anyway and there is no risk of braking something easily. ;) 

On the others hand - I would move '49' number to const of this class and describe it - why it's 49.